examples/auctions/simple_open_auction.v.py

```diff
@@ -0,0 +1,76 @@
+# Open Auction
+
+# Auction params
+# Beneficiary recieves money from the highest bidder
+beneficiary: public(address)
+auction_start: public(timestamp)
+bidding_time: public(num(sec))
+
+# Current state of auction
+highest_bidder: public(address)
+highest_bid: public(num(wei))
+
+# Allowed withdrawals of previous bids
+pending_returns:(num(wei)[address])
+
+# Set to true at the end, disallows any change
+ended: public(bool)
+
+# Create a simple auction with `_bidding_time`
+# seconds bidding time on behalf of the
+# beneficiary address `_beneficiary`.
+def __init__(_beneficiary: address, _bidding_time: num(sec)):
+    self.beneficiary = _beneficiary
+    self.bidding_time = _bidding_time
+    self.auction_start = block.timestamp
+
+# Bid on the auction with the value sent
+# together with this transaction.
+# The value will only be refunded if the
+# auction is not won.
+@payable
+def bid():
```

Comment:
If someone had bid previously, and they were overbid but decided to bid again, they now carry an extra balance in `pending_returns` in addition to what they are now bidding. Can you ADD the value sent in the bid (e.g. new bid = old bid + msg.value, asserting that new bid is over the highest bidding amount and removing the old bid from `pending_returns`) so the users wouldn't have to tie up an exponentially increasing amount of wei depending on the amount of bids there are, only to have that excess wei returned when the auction is over? Do you follow what I am saying?